packages/react/src/components/pickers/Suggestions/SuggestionsItem.styles.ts

```diff
@@ -115,7 +124,7 @@ export function getStyles(props: ISuggestionsItemStyleProps): ISuggestionsItemSt
       suggested && {
         selectors: {
           ':hover, :active': {
-            background: palette.neutralTertiary,
+            background: `${palette.neutralTertiary} !important`,
```

Comment:
It's a best practice not to use `!important` unless absolutely necessary (which is generally only when you need to modify styles you're not in control of).

For this, it'd be better to have the `.${IsFocusVisibleClassName} &` selector + style block here, so you don't need `!important` to override the button background color.

Reply:
Ah something did feel fishy about that `!important` - thanks! I've made the change